ons_alpha/bulletins/models.py

```diff
@@ -103,9 +105,10 @@ def is_latest(self):
         return self.pk == latest_id
 
     @cached_property
-    def toc(self):
+    def toc(self) -> List[Dict[str, str]]:
         items = [{"url": "#summary", "text": "Summary"}]
-        for block in self.body:  # pylint: disable=not-an-iterable
+        body = self.body  # pylint: disable=not-an-iterable
+        for block in body:
```

Comment:
The linter doesn't seem to like these lines, but I'm not completely sure why. `self.body` _is_ an iterable, so it's likely a type inference issue. I think changing this to `for block in self.body`, confirming it works, and then doing whatever you need to do appease `pylint` (which might simply be ignoring specific errors.

I'd suggest leaving these lines as-is, as `pylint` was clearly ok with them before.